### PR TITLE
PAB-3526 - Release note for Apama /health changes

### PR DESCRIPTION
**Pull request by rthrippleton**

### CODE REVIEW

File: content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md [hunk 1]
```diff
@@ -73,3 +73,7 @@ The **Latest Updated** mode of the
 [Combiner](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fre_AnaBui_block_reference_Flow_Manipulation_Combiner.html) block, which has been added in
 [release 10.16.0](/release-10-16-0/streaming-analytics-10-16-0/),
 has been renamed to **Latest Input**. This was done so that you can better distinguish it from the **Latest Changed** mode.
+
+### Changes to microservice health endpoint
```
Comment (kailol-sag) [inline content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md:77]: I think this info should be moved into the "Important announcements" in content/release-10-17-0/announcements-10-17-0.md

Comment (rthrippleton) [reply content/release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md]: Good point, it fits with everything else we put there, even though it's minor.

20cb144

File: content/release-10-17-0/announcements-10-17-0.md [hunk 2]
```diff
@@ -49,3 +49,7 @@ See also the announcement in the Streaming Analytics release notes for
 Existing blocks that use the version 1 API must be migrated to use the version 2 API.
 See [Migrating input and output blocks to the version 2 API](https://github.com/SoftwareAG/apama-analytics-builder-block-sdk/blob/rel/10.17.0.x/doc/150-MigrateInputOutputBlocks.md)
 in the Analytics Builder Block SDK documentation on GitHub for more details.
+
+#### Changes to microservice health endpoint
```
Comment (kailol-sag) [inline content/release-10-17-0/announcements-10-17-0.md:53]: Change the heading to ##### (that is, one more #).

File: content/release-10-17-0/announcements-10-17-0.md [hunk 3]
```diff
@@ -49,3 +49,7 @@ See also the announcement in the Streaming Analytics release notes for
 Existing blocks that use the version 1 API must be migrated to use the version 2 API.
 See [Migrating input and output blocks to the version 2 API](https://github.com/SoftwareAG/apama-analytics-builder-block-sdk/blob/rel/10.17.0.x/doc/150-MigrateInputOutputBlocks.md)
 in the Analytics Builder Block SDK documentation on GitHub for more details.
+
+##### Changes to microservice health endpoint
+
+For reasons of security and performance, the REST endpoint `/service/cep/health` no longer returns a comprehensive list of status values. All of the same information is still available from REST endpoints under `/service/cep/diagnostics/...`.
```
Comment (kailol-sag) [inline content/release-10-17-0/announcements-10-17-0.md:55]: Wondering if we really need the 3 dots after "diagnostics/". I'd leave them out.

Comment (rthrippleton) [reply content/release-10-17-0/announcements-10-17-0.md:55]: My thinking was that I don't want to give the impression that there is a specific endpoint called `/service/cep/diagnostics`, and I'm actually trying to say "every endpoint that starts with that".

If you still think I should remove the '...' though, I'm fine.

Comment (kailol-sag) [reply content/release-10-17-0/announcements-10-17-0.md:55]: That's why I did not remove the / from my suggestion ;-)
However, if @BeateRixen doesn't mind, we can keep the 3 dots.

Comment (BeateRixen) [reply content/release-10-17-0/announcements-10-17-0.md:55]: Fine with me :-).

**Review by kailol-sag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by bph-sag — APPROVED** [review]

**Pull request merged**